mlbench_core/cli/cli.py

```diff
@@ -130,17 +130,37 @@ def run(name, num_workers, gpu, light, dashboard_url):
         # run custom image
         image = click.prompt("Image", type=str)
         image_command = click.prompt("Command", type=str)
-        run_on_all = click.confirm(
-            "Run command on all nodes (otherwise just first node)"
-        )
+        # run_on_all = click.confirm(
+        #     "Run command on all nodes (otherwise just first node)"
+        # )
         benchmark = {
             "custom_image_name": image,
             "custom_image_command": image_command,
-            "custom_image_all_nodes": run_on_all,
+            # "custom_image_all_nodes": run_on_all,
         }
     else:
         benchmark = {"image": images[selection]}
 
+    # Backend Prompt
+    text_prompt = "Backend: \n\n"
+    text_prompt += "\n".join("[{}]\t{}".format(i, t) for i, t in enumerate(MLBENCH_BACKENDS))
+    text_prompt += "\n[{}]\tCustom Backend".format(len(MLBENCH_BACKENDS))
+    text_prompt += "\n\nSelection"
+
+    selection = click.prompt(
+        text_prompt, type=click.IntRange(0, len(MLBENCH_BACKENDS)), default=0
```

Comment:
Something i've thought about (also for the other prompts), it'd be awesome if the cli remembered the last input as default. If you'd like you can add this in this PR or I can make a new issue for it.

Reply:
So it would remember what arguments were used the last time the command was run ?